submissions/cynthiaNwakaeme/CSS/style.css

```diff
@@ -0,0 +1,286 @@
+    font-weight: bolder;
+    color: goldenrod;
+}
+.form-details button{
+    width: 25%;
+    height: 40px;
+    border-radius: 5px;
+    background-color: #9d426b;
+    color: #fff;
+    margin: 0 165px;
+    border: none;
+    font-size: 18px;
+}
+
+/* THE FOOTER */
+footer{
+    overflow: hidden;
+    background-color: #9d426b;
+    padding:20px 0 0 50px;
+    color: white;
+}
+.footer-box,
+.footer-box-2,
+.footer-box-3{
+    float: left;
+    width: 33%;
+}
+.footer-box ul li,
+.footer-box-3 ul li
+{
+    list-style: none;
+    margin-bottom: 8px;
+}
+.footer-box ul li a{
+    text-decoration: none;
+    display: inline;
+    color: white;  
+}
+.footer-box-2 button[type="submit"]{
+    height: 30px;
+    padding: 0 5%;
+    font-size: 12px;
+    color: #9d426b;
+    font-weight: bold;
+}
+.footer-box-2 input[type="email"]{
+    height: 20px;
+    /* width: 200px; */
+    padding: 3px 18px 3px ;
+    font-size: 11px; 
+}
+.footer-box-3 ul li{
+    display: inline-flex;
+    padding:10px ;
+}
+.footer-box-3 ul li img{
+    width: 35px;
+}
+footer p{
+    text-align: center;
+    display: block;
+}
+/* HANDLING RESPONSIVENESS / RESOLUTIONS */
+
+@media screen and (min-width: 709px) and (max-width: 768px) {
+    #main,
+    #aside{
+        float:none;
+        width: 70%;
+        padding-bottom: 10px;
+    }
+    #main{
+        text-align: center;
+    }
+    #aside fieldset{
+        margin-left: 60px;
+    }
+    .form-details input[type="checkbox"]{
+        margin-left: 20%; 
+    }
+    .footer-box,
+    .footer-box-2{
+        width: 50%;
+    }
+    .footer-box-3{
+        float: none;
+        width: 50%;  
+    }
+    .footer-box-3 ul li{
+        
+        display: inline;
+    }
+    
+}
+@media screen and (max-width: 708px){
+     
+    header #nav-branding,
+    header nav,
+    header .nav-container  nav li{
+        float:none;
+        text-align: center;
+        width: 100%;
+        padding:0 3%;
+    }
+    .container{
+        padding:0 3%;
+    }
+    #main,
+    #aside{
+        float:none;
+        padding-bottom: 15px;
+    }
+    #main{
+        text-align: center;
+        width: 85%;
+        margin-left: 25px;
+    }
+    #aside{
+        width: 80%;
+        margin-left: 20px;
+        margin-right: 50%;
+    }
+    .form-details input[type="text"],
+    .form-details input[type="tel"],
+    .form-details input[type="email"],
+    .form-details input[type="password"]
+    {
+    width: 95%;
+    height: 25px;
+    border: 1px solid rgb(129, 129, 129);
+    border-radius: 8px;
+    margin-bottom: 3px;
+    
+    }
+    .form-details input[type="checkbox"]{
+        margin-left: 30px; 
+    }
+    .form-details button{
+        width: 90px;
+        padding: 5px;
+        height: 40px;
+        border-radius: 5px;
+        color: #fff;
+        margin: 0 90px;
+        border: none;
+        font-size: 18px;
+    }
+    
+    .footer-box,
+    .footer-box-2,
+    .footer-box-3{
+        float: none;
```

Comment:
There isn't an attribute `float: none` you can remove it.

Reply:
Ma please i need help on how to remove the initialized float for responsiveness.That's the only information I got online.
I am stuck already.

Reply:
Alright.  You can leave as well @Cynthia-Berry